test/masq.test.js

```diff
@@ -7,7 +7,11 @@ window.crypto = require('@trust/webcrypto')
 
 const Masq = require('../src')
 const MasqAppMock = require('./mockMasqApp')
-const config = require('../config/config')
+const localhostConfig = require('../config/config.localhost.json')
```

Comment:
Don't you prevers config.test.json instead ?

Reply:
changed localhost to test